templates/genie.yaml

```diff
@@ -15,8 +15,27 @@ Parameters:
   MutationImage:
     Description: The Genie mutation processing docker container image
     Type: String
+  Submitter:
+    Description: AWS account submitting batch job
+    Type: String
 
 Resources:
+  BatchSubmitRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Action:
+          - batch:SubmitJob
+          - sts:AssumeRole
+          Resource:
+          - arn:aws:batch:us-east-1:055273631518:job-definition/genie-*
```

Comment:
you can use the Pseduo parameters `AWS::Region` and `AWS::AccountId` instead, https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html, for